conf/script/ha_setup

```diff
@@ -60,6 +60,9 @@ config() {
     # Decision Maker conf
```

Comment:
in future we may need to improve setup to do complex work

like changing to python
@ajaysrivas  @indrajitzagade

Reply:
ok